model/riscv_insts_aext.sail

```diff
@@ -48,9 +48,14 @@ mapping clause encdec = LOADRES(aq, rl, rs1, size, rd)
 
 
 /* We could set load-reservations on physical or virtual addresses.
- * For now we set them on virtual addresses, since it makes the
- * sequential model of SC a bit simpler, at the cost of an explicit
- * call to load_reservation in LR and cancel_reservation in SC.
+ * However the reservation is architecturally defined in terms of physical
```

Comment:
Is this actually the case? The LR and SC section of the unpriv spec never uses the word physical. "a set of bytes that subsumes the bytes in the addressed word." seems intentionally fuzzy to allow a range of implementations.

Reply:
OTOH, when I search the priv spec: "For implementations with both page-based virtual memory and the "A" standard extension, the LR/SC
reservation set must lie completely within a single base physical page (i.e., a naturally aligned 4 KiB
physical-memory region)."

Reply:
This is based on [this comment](https://github.com/riscv/sail-riscv/issues/360#issuecomment-1890399132); maybe the language is a little too strong for something in an second hand email... Maybe I should just remove that comment? Though the quote you found does indirectly support it.

Reply:
Ok comment removed.